client/signup/videopress-step-wrapper/style.scss

```diff
@@ -237,6 +237,7 @@ body.is-section-signup .videopress-step-wrapper {
 	}
 
 	.auth-form__separator-text {
+		background: none;
```

Comment:
That's a bit suspicious, the separator test doesn't have a BG, what is setting it then reverting it?